elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -71,4 +71,8 @@
 	@Message(id = ES_BACKEND_MESSAGES_START_ID + 9,
 			value = "'%1$s' is not a remote analyzer. It will be ignored")
 	void analyzerIsNotRemote(String name);
+
+	@Message(id = ES_BACKEND_MESSAGES_START_ID + 10,
+			value = "Elasticsearch connection time-out; check the cluster status, it should be 'green';\n Request:\n========\n%1$sResponse:\n=========\n%2$s" )
```

Comment:
Get another editor than Notepad and it will work ;)

Kidding aside, the request and response will contain line feeds anyways (JSON structures). Also I expect people to use tools such as Splunk, Logstash etc. when analyzing production logs. To me, it's important to get out the required context in a sensible way, so I don't think we should let stop ourselves by people stuck with their tools in the 90ies.